apps/backoffice-v2/src/pages/Entity/components/CallToActionLegacy/CallToActionLegacy.tsx

```diff
@@ -0,0 +1,314 @@
+import { AnimatePresence } from 'framer-motion';
+import { ComponentProps, FunctionComponent } from 'react';
+
+import { Send, X } from 'lucide-react';
+import { Button } from '../../../../common/components/atoms/Button/Button';
+import { Input } from '../../../../common/components/atoms/Input/Input';
+import { Select } from '../../../../common/components/atoms/Select/Select';
+import { SelectContent } from '../../../../common/components/atoms/Select/Select.Content';
+import { SelectItem } from '../../../../common/components/atoms/Select/Select.Item';
+import { SelectTrigger } from '../../../../common/components/atoms/Select/Select.Trigger';
+import { SelectValue } from '../../../../common/components/atoms/Select/Select.Value';
+import { MotionButton } from '../../../../common/components/molecules/MotionButton/MotionButton';
+import { Dialog } from '../../../../common/components/molecules/Dialog/Dialog';
+import { capitalize } from '../../../../common/utils/capitalize/capitalize';
+import { ctw } from '../../../../common/utils/ctw/ctw';
+import { DocumentPicker } from '../DirectorsCallToAction/components/DocumentPicker';
+import { useCallToActionLegacyLogic } from '@/pages/Entity/components/CallToActionLegacy/hooks/useCallToActionLegacyLogic/useCallToActionLegacyLogic';
+import { ICallToActionLegacyProps } from './interfaces';
+
+const motionProps: ComponentProps<typeof MotionButton> = {
+  exit: { opacity: 0, transition: { duration: 0.2 } },
+  initial: { y: 10, opacity: 0 },
+  transition: { type: 'spring', bounce: 0.3 },
+  animate: { y: 0, opacity: 1, transition: { duration: 0.2 } },
+};
+
+export const CallToActionLegacy: FunctionComponent<ICallToActionLegacyProps> = ({ value }) => {
```

Comment:
We will no longer use this component once dialogs are separated from buttons and the directors logic is separated out. (adding a task)